sa4u_z3/main.py

```diff
@@ -388,6 +388,60 @@ def TERM_signal_handler(sig_num: int, _frame):
     sys.exit()
 
 
+def handle_serialized_TU(tu: SerializedTU):
```

Comment:
Maybe we should rename this to something like `get_z3_assertions_from_stu`

Reply:
Done